### DISCUSSION TITLE
Use separate files for imports and actual CSS it isn't working

### DISCUSSION THREAD

**Discussion by carveybunt** [tailwindlabs/tailwindcss#7453] [Help]
* my `tailwind.config.js` code:
```js
module.exports = {
  prefix: 'tw-',
  important: true,
  content: ['./packages/components/button/**/*.{ts,vue}'],
  theme: {
    extend: {},
  },
  plugins: [],
}
```
* my `postcss.config.js` code:
```js
module.exports = {
  plugins: [
    require("postcss-import"),
    require("tailwindcss"),
    require("autoprefixer"),
  ],
}
```
* my `tailwind.css` code:
```css
@import "tailwindcss/base";
@import "tailwindcss/components";
@import "tailwindcss/utilities";

@import "./te-main.css";
```
* my `te-main.css` code:
```css
.tw-text-cc-500 {
  @apply tw-text-indigo-600;
}
```
It isn't working when I running `npx tailwindcss -i tailwind.css -o tailwinds.css --watch`. The file of `tailwind.css` didn't generate the class of `tw-text-cc-500`. Could someone help me, please?

**Comment by sntran** [comment]
Silly question, but `tw-text-cc-500` class is used in `./packages/components/button/**/*.{ts,vue}`, right?

**Reply by carveybunt** [reply]
> Silly question, but `tw-text-cc-500` class is used in `./packages/components/button/**/*.{ts,vue}`, right?

thanks a lot, I already soved that.

**Comment by adamwathan** [comment]
Hey! Make sure you add the `--postcss` flag when running the CLI or it won't pick up your PostCSS config, and the import won't work:

```
npx tailwindcss -i tailwind.css -o tailwinds.css --postcss --watch
```
